pkg/controllers/clusterexternalsecret/clusterexternalsecret_controller.go

```diff
@@ -152,6 +152,9 @@ func (r *Reconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Resu
 
 	failedNamespaces := r.deleteOutdatedExternalSecrets(ctx, namespaceList, esName, clusterExternalSecret.Name, clusterExternalSecret.Status.ProvisionedNamespaces)
 
+	condition := NewClusterExternalSecretCondition(failedNamespaces)
+	SetClusterExternalSecretCondition(&clusterExternalSecret, *condition)
+
```

Comment:
Why do we need the two lines above? 🤔

Reply:
Thank you for the review and suggestion @shuheiktgw . I have removed these lines.